Cesium3DTilesSelection/src/TilesetLoadIonAssetEndpoint.cpp

```diff
@@ -149,12 +210,11 @@ Tileset::LoadIonAssetEndpoint::Private::mainThreadHandleResponse(
 
       for (const rapidjson::Value& attribution :
            attributionsIt->value.GetArray()) {
-
+        AssetEndpointAttribution endpointAttribution;
         const auto html = attribution.FindMember("html");
         if (html != attribution.MemberEnd() && html->value.IsString()) {
-          tileset._tilesetCredits.push_back(
-              tileset.getExternals().pCreditSystem->createCredit(
-                  html->value.GetString()));
+          endpointAttribution.html = html->value.GetString();
+          endpoint.attributions.push_back(endpointAttribution);
```

Comment:
Slightly cleaner / more performant to do:

```
AssetEndpointAttribution& endpointAttribution = endpoint.attributions.emplace_back();
endpointAttribution.html = html->value.GetString();
```

Reply:
Thanks for the tip.